roseau/load_flow/models/loads/loads.py

```diff
@@ -53,21 +53,32 @@ def __init__(self, id: Id, bus: Bus, *, phases: str | None = None) -> None:
                 The phases of the load. A string like ``"abc"`` or ``"an"`` etc. The bus phases are
                 used by default. The order of the phases is important. For a full list of supported
                 phases, see the class attribute :attr:`allowed_phases`. All phases of the load must
-                be present in the phases of the connected bus. Multiphase loads are allowed to have
-                a floating neutral (i.e. they can be connected to buses that don't have a neutral).
+                be present in the phases of the connected bus. Multiphase loads are allowed to be
+                connected to buses that don't have a neutral if ``connect_neutral`` is not set to
+                ``True``.
+
+            connect_neutral:
+                Specifies whether the load's neutral should be connected to the bus's neutral or
+                left floating. By default, the load's neutral is connected when the load has a
+                neutral. If the bus does not have a neutral, the load's neutral is left floating
+                by default. To override the default behavior, pass an explicit ``True`` or ``False``.
         """
         if type(self) is AbstractLoad:
             raise TypeError("Can't instantiate abstract class AbstractLoad")
         super().__init__(id)
```

Comment:
Maybe an error should be raised for some cases, for example if `connect_neutral` is True, but there are no neural in the bus or in the phases.